Library/ViewModels/PledgeViewModel.swift

```diff
@@ -1,21 +1,44 @@
 import Foundation
 import KsApi
+import PassKit
 import Prelude
 import ReactiveSwift
 
+public typealias StripeConfigurationData = (merchantIdentifier: String, publishableKey: String)
+public typealias BackingData = (
+  project: Project, reward: Reward, pledgeAmount: Double,
+  selectedShippingRule: ShippingRule?
+)
+public typealias PaymentAuthorizationData = (
+  project: Project, reward: Reward, pledgeAmount: Double,
+  selectedShippingRule: ShippingRule?, merchantIdentifier: String
+)
+public typealias PKPaymentData = (displayName: String, network: String, transactionIdentifier: String)
+
 public protocol PledgeViewModelInputs {
+  func applePayButtonTapped()
   func configureWith(project: Project, reward: Reward)
+  func paymentAuthorizationDidAuthorizePayment(
+    paymentData:
+    (displayName: String?, network: String?, transactionIdentifier: String)
```

Comment:
This can be on the same line as the param name.

Reply:
SwiftFormat is doing all of this.